packages/core/src/stylable-transformer.ts

```diff
@@ -578,7 +586,10 @@ export class StylableTransformer {
                             : node.nodes
                     ) as Selector[];
                     const nestedContext = context.createNestedContext(innerSelectors);
-                    this.scopeSelectorAst(nestedContext);
+                    const transformedSelectors = this.scopeSelectorAst(nestedContext);
+                    for (let i = 0; i < transformedSelectors.length; i++) {
```

Comment:
for of?

Reply:
I need the index

Reply:
.entries :D

Reply:
I like the plain loop here. I'm keeping it.

Reply:
you do you